CHANGELOG.md

```diff
@@ -58,6 +58,8 @@ For details about compatibility between different releases, see the **Commitment
 
 ### Changed
 
+- Set default page limit to 100 on AS and NS List RPCs
+
```

Comment:
Can you please add a note on what are the implications of this value and how an operator can adjust it?